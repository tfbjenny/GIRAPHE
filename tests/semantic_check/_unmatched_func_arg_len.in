int foo(int a, int b) {
    return a + b;
}

foo(1,2,3);