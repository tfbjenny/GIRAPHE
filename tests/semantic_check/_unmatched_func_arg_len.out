Uh.. This args length does not match in function call: main.foo :(
